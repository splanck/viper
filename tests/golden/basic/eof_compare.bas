' eof_compare.bas
OPEN "tmp_eof.txt" FOR OUTPUT AS #1
PRINT #1, "hello"
CLOSE #1

OPEN "tmp_eof.txt" FOR INPUT AS #1
DIM LINE$
DO WHILE EOF(#1) = 0
  LINE INPUT #1, LINE$
  PRINT "READ:"; LINE$
LOOP
PRINT "EOF now ="; EOF(#1)
CLOSE #1
