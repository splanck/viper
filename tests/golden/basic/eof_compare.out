READ:hello
EOF now = -1
